src/airport.js

```diff
@@ -1 +1,49 @@
+class Airport {
+    hangar;
+    #maxCapacity;
+    weather;
 
+    constructor(plane = [], capacity = 3, weather = 'Clear') {
+        this.hangar = plane;
+        this.#maxCapacity = capacity;
+        this.weather = weather;
+    }
+
+    getWeather() {
+        return this.weather;
+    }
+
+    changeWeather(weather) {
+        weather === 'Stormy' ? this.weather = 'Stormy' : weather === 'Clear' ? this.weather = 'Clear' : console.log(`Please enter valid weather conditions. Valid conditions are 'Clear' or 'Stormy'.`);
```

Comment:
This ternary statement might be too complicated, may have been clearer to just use if... else to make code easier to read.